Annex60/Fluid/Chillers/Carnot_TEva.mo

```diff
@@ -160,6 +156,24 @@ Annex60.Fluid.HeatPumps.Examples.Carnot_TCon</a>.
 revisions="<html>
 <ul>
 <li>
+January 2, 2017, by Filip Jorissen:<br/>
+Removed parameters
+<code>effInpEva</code> and <code>effInpCon</code>
+and updated documentation.
+This is for
+<a href=\"https://github.com/iea-annex60/modelica-annex60/issues/497\">
+issue 497</a>.
+</li>
+<li>
+August 8, 2016, by Michael Wetter:<br/>
```

Comment:
I am a bit confused whether this date is intentional chosen or if you forgot to edit it?